src/preact/component/value-ref.ts

```diff
@@ -1,12 +1,14 @@
+import type {RefObject} from 'preact';
```

Comment:
@samouri is it okay to do type imports from preact directly? Do we need to re-export types from #preact like we do with other symbols?

Reply:
(I had the same question)

Reply:
Fine to use `import type`  from preact since it all gets erased during compilation.